src/status_im/chat/models.cljs

```diff
@@ -56,6 +56,14 @@
   [{:keys [current-chat-id] :as db} ui-element]
   (update-in db [:chat-ui-props current-chat-id ui-element] not))
 
+(defn join-time-messages-checked
```

Comment:
I would add a description of what these fields are used for, as it might not be immediately clear

Reply:
will do

Reply:
done